sample-btcd.conf

```diff
@@ -156,6 +156,9 @@
 ; Add additional checkpoints. Format: '<height>:<hash>'
 ; addcheckpoint=<height>:<hash>
 
+; Add comments to the User Agent that is advertised to peers.
+; Should not include characters '/', ':', '(' and ')'.
```

Comment:
Since this produces an error in the code, it should say "Must not include...".  Should implies it is optional to do so which is not the case.